ansible_collections/arista/avd/examples/single-dc-l3ls/deploy-cvp.yml

```
@@ -0,0 +1,12 @@
+---
+- name: Deploy Configurations to Devices Using CloudVision Portal # (1)!
+  hosts: CLOUDVISION
+  gather_facts: false
+  connection: local
+  tasks:
+
+    - name: Deploy Configurations to CloudVision # (2)!
+      ansible.builtin.import_role:
+        name: arista.avd.eos_config_deploy_cvp
+      vars:
+        cv_collection: v3 # (3)!
```

When testing this the playbook fails because the CVP server has no `fabric_name` set, and the default `root_container` and `configlet_prefix` use the `fabric_name`. So either set the `fabric_name` or override the other defaults to not use `fabric_name`.
I tested with this:
```suggestion
        cv_collection: v3 # (3)!
        fabric_name: FABRIC
```

---

Alternatively you can move the setting of fabric_name to the `all.yml` group var.

---

Ill add this to the cvp playbook, thanks Claus